cli/Sources/TuistLoader/SwiftPackageManager/PackageInfoMapper.swift

```diff
@@ -1395,25 +1395,47 @@ extension PackageInfo.Target {
     /// The path used as base for all the relative paths of the package (e.g. sources, resources, headers)
     func basePath(packageFolder: AbsolutePath) async throws -> AbsolutePath {
         let fileSystem = FileSystem()
+
         if let path {
             return packageFolder.appending(try RelativePath(validating: path))
-        } else {
-            let predefinedDirectories: [String]
-            switch type {
-            case .test:
-                predefinedDirectories = PackageInfoMapper.predefinedTestDirectories
-            default:
-                predefinedDirectories = PackageInfoMapper.predefinedSourceDirectories
+        }
+
+        let predefinedDirectories = type == .test
+            ? PackageInfoMapper.predefinedTestDirectories
+            : PackageInfoMapper.predefinedSourceDirectories
+
+        for dir in predefinedDirectories {
```

Comment:
```suggestion
        for directory in predefinedDirectories {
```

nit: generally prefer long-form versions of variables